files/default/sql/ddl/updates/0.7.0.sql

```diff
@@ -26,4 +26,4 @@ ALTER TABLE `hopsworks`.`ops_log` MODIFY COLUMN `inode_id` BIGINT(20);
 ALTER TABLE `hopsworks`.`meta_log` MODIFY COLUMN `meta_pk2` BIGINT(20);
 ALTER TABLE `hopsworks`.`meta_log` MODIFY COLUMN `meta_pk3` BIGINT(20);
 
-ALTER TABLE `hopsworks`.`conda_commands` ADD COLUMN `install_jupyter` tinyint(1) NOT NULL DEFAULT '0';
\ No newline at end of file
+ALTER TABLE `hopsworks`.`conda_commands` ADD COLUMN `install_jupyter` tinyint(1) NOT NULL DEFAULT '0';
```

Comment:
@Limmen Please don't modify ddl files for older releases, otherwise they will be re-remplated by chef with a different checksums and will break flyway migrations

Reply:
@SirOibaf Yeah good catch, it was unintentional, must have added a space or something when moving featurestore DDL/DML from 0.7.0 to 0.8.0